WordPress/src/main/res/values/strings.xml

```diff
@@ -3339,6 +3339,7 @@
     <string name="hpp_title">Choose a design</string>
     <string name="hpp_subtitle">Pick your favorite homepage layout. You can edit and customize it later.</string>
     <string name="hpp_choose_button">Choose</string>
+    <string name="hpp_choose_and_create_site">Create site</string>
```

Comment:
@antonis Unfortunately, since that PR was merged into `release/19.7` and thus _after_ the code freeze, it wasn't included in the frozen strings sent to our Translation Vendor last Tuesday for this version.

Since I caught this up today by chance while working on the release branch, and I've decided to update the frozen strings file to include that new one (via: d55cc88), so that at least it gets imported into GlotPress ASAP and hopefully translated in the Mag16 by the community in time, even if our translator vendor (who usually ensure Mag16 coverage) didn't get that one in time. 🤞 

For the record and for next time, please be sure to ping your Release Manager when you need to update string copies on PRs done post-code-freeze and targeting the `release/*` branch 🙇